extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/tracing/intrumentation/vertx/wrapper/DevModeOpenTelemetryWrapper.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.opentelemetry.runtime.tracing.intrumentation.vertx.wrapper;
+
+import io.opentelemetry.api.GlobalOpenTelemetry;
+import io.opentelemetry.api.OpenTelemetry;
+
+public class DevModeOpenTelemetryWrapper implements OpenTelemetryWrapper {
+    private OpenTelemetry openTelemetry;
+
+    public DevModeOpenTelemetryWrapper() {
+        this.openTelemetry = GlobalOpenTelemetry.get();
+    }
+
+    @Override
+    public OpenTelemetry get() {
+        return openTelemetry;
+    }
+
+    @Override
+    public Boolean resetIfChanged() {
+        OpenTelemetry currentOpenTelemetry = GlobalOpenTelemetry.get();
+        if (this.openTelemetry != null && this.openTelemetry.equals(currentOpenTelemetry)) {
```

Comment:
This should probably just be an `==`, as it is the identity that we care about.

Reply:
obsolete